ipynbs/Chasing_Platypus_via_MG-RAST.ipynb

```diff
@@ -270,8 +16,368 @@
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython2",
    "version": "2.7.6"
-  }
+  },
+  "name": "",
+  "signature": "sha256:08500e8588bd6415c7eb54d38462cde68383678e817fda6d7d608c1cb34519a3"
  },
- "nbformat": 4,
- "nbformat_minor": 0
+ "nbformat": 3,
+ "nbformat_minor": 0,
+ "worksheets": [
+  {
+   "cells": [
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "----------------------------------------------------------------------------\n",
+      "Copyright (c) 2015--, platypus development team.\n",
+      "\n",
+      "Distributed under the terms of the BSD License.\n",
+      "\n",
+      "The full license is in the file [COPYING.txt](https://raw.githubusercontent.com/biocore/Platypus-Conquistador/master/COPYING.txt), distributed with this software.\n",
+      "\n",
+      "----------------------------------------------------------------------------"
+     ]
+    },
+    {
+     "cell_type": "heading",
+     "level": 2,
+     "metadata": {},
+     "source": [
+      "Chasing the Platypus via MG-RAST"
+     ]
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "This tutorial covers downloading all public MG-RAST datasets; parsing those files to search for a specific taxa of interest, in our case Platypus (Ornithorhynchus); and finding the geographycal location of the samples that match those samples, via Google. Note that you need [BeautifulSoup](http://www.crummy.com/software/BeautifulSoup/#Download/) installed."
+     ]
+    },
+    {
+     "cell_type": "heading",
+     "level": 2,
+     "metadata": {},
+     "source": [
+      "Downloading all public MG-RAST datasets"
+     ]
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "<b>Before you continue: </b>This step is extremelly slow (~24 hrs.) and network intensive. Thus to facilitate this step you could download the files from April 22, 2015 [ftp://ftp.microbio.me/pub/platypus/mgrast_reports_042215.tgz](ftp://ftp.microbio.me/pub/platypus/mgrast_reports_042215.tgz). If you place them in the same folder that you have this IPython Notebook the code will simply download the latests samples.\n",
```

Comment:
Since this is MarkDown, there's no need to do `<b>`, you can just simply do `**bold**`.